apm-agent-plugins/apm-httpclient11-plugin/src/main/java/co/elastic/apm/agent/httpclient/HttpClientHelper.java

```diff
@@ -0,0 +1,22 @@
+package co.elastic.apm.agent.httpclient;
+
+import co.elastic.apm.agent.bci.VisibleForAdvice;
+
+import javax.annotation.Nullable;
+
+@VisibleForAdvice
```

Comment:
When using indy plugins, this annotation is not needed. You can even make the class package-private if you want to.

Reply:
deleted